care/users/api/serializers/user.py

```diff
@@ -1,7 +1,9 @@
 from django.contrib.auth import get_user_model
 from django.contrib.auth.hashers import make_password
-from rest_framework import serializers
+from django.db import transaction
+from rest_framework import exceptions, serializers
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Unable to import 'rest_framework'](https://app.codacy.com/gh/coronasafe/care/pullRequest?prid=5345345)